src/components/Kv/KvFlag2.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+	<div :class="`kv-flag kv-flag--${aspectRatio}`">
+		<div class="kv-flag__wrapper">
+			<template v-if="fromSprite">
+				<div
+					class="kv-flag__img kv-flag__img--sprite"
+					:style="{ backgroundPositionY: spriteYPosition + '%' }"
+				></div>
+			</template>
+			<template v-else>
+				<component
+					:is="flagSVG"
+					class="kv-flag__img"
+				/>
+			</template>
+		</div>
+	</div>
+	<!-- TODO: Maybe add the country name. Either as screenreader only text or use KvTooltip. -->
+</template>
+
+<script>
+const COUNTRY_LIST = ['ad', 'ae', 'af', 'ag', 'ai', 'al', 'am', 'ao', 'aq', 'ar', 'as', 'at', 'au', 'aw', 'ax', 'az', 'ba', 'bb', 'bd', 'be', 'bf', 'bg', 'bh', 'bi', 'bj', 'bl', 'bm', 'bn', 'bo', 'bq', 'br', 'bs', 'bt', 'bv', 'bw', 'by', 'bz', 'ca', 'cc', 'cd', 'cf', 'cg', 'ch', 'ci', 'ck', 'cl', 'cm', 'cn', 'co', 'cr', 'cu', 'cv', 'cw', 'cx', 'cy', 'cz', 'de', 'dj', 'dk', 'dm', 'do', 'dz', 'ec', 'ee', 'eg', 'eh', 'er', 'es-ca', 'es-ga', 'es', 'et', 'eu', 'fi', 'fj', 'fk', 'fm', 'fo', 'fr', 'ga', 'gb-eng', 'gb-nir', 'gb-sct', 'gb-wls', 'gb', 'gd', 'ge', 'gf', 'gg', 'gh', 'gi', 'gl', 'gm', 'gn', 'gp', 'gq', 'gr', 'gs', 'gt', 'gu', 'gw', 'gy', 'hk', 'hm', 'hn', 'hr', 'ht', 'hu', 'id', 'ie', 'il', 'im', 'in', 'io', 'iq', 'ir', 'is', 'it', 'je', 'jm', 'jo', 'jp', 'ke', 'kg', 'kh', 'ki', 'km', 'kn', 'kp', 'kr', 'kw', 'ky', 'kz', 'la', 'lb', 'lc', 'li', 'lk', 'lr', 'ls', 'lt', 'lu', 'lv', 'ly', 'ma', 'mc', 'md', 'me', 'mf', 'mg', 'mh', 'mk', 'ml', 'mm', 'mn', 'mo', 'mp', 'mq', 'mr', 'ms', 'mt', 'mu', 'mv', 'mw', 'mx', 'my', 'mz', 'na', 'nc', 'ne', 'nf', 'ng', 'ni', 'nl', 'no', 'np', 'nr', 'nu', 'nz', 'om', 'pa', 'pe', 'pf', 'pg', 'ph', 'pk', 'pl', 'pm', 'pn', 'pr', 'ps', 'pt', 'pw', 'py', 'qa', 're', 'ro', 'rs', 'ru', 'rw', 'sa', 'sb', 'sc', 'sd', 'se', 'sg', 'sh', 'si', 'sj', 'sk', 'sl', 'sm', 'sn', 'so', 'sr', 'ss', 'st', 'sv', 'sx', 'sy', 'sz', 'tc', 'td', 'tf', 'tg', 'th', 'tj', 'tk', 'tl', 'tm', 'tn', 'to', 'tr', 'tt', 'tv', 'tw', 'tz', 'ua', 'ug', 'um', 'un', 'us', 'uy', 'uz', 'va', 'vc', 've', 'vg', 'vi', 'vn', 'vu', 'wf', 'ws', 'xk', 'ye', 'yt', 'za', 'zm', 'zw']; // eslint-disable-line max-len
+const SPRITE_FLAG_WIDTH = 32; // Number of px wide that the sprite PNG is.
+
+export default {
+	props: {
+		country: {
+			type: String,
+			required: true,
+		},
+		aspectRatio: {
+			type: String,
+			default: '4x3', // 4x3 or 1x1
+		},
+		fromSprite: {
+			type: Boolean,
+			default: false
+		}
+	},
+	computed: {
+		flagSVG() {
+			return () => import(`~/flag-icon-css/flags/${this.aspectRatio}/${this.country}.svg`);
```

Comment:
This is how the inlined svgs are handled. Same ideas as KvIcon.

Reply:
I initially thought we might want a fallback image incase this fails for some reason. However, my main concern is that the layout isn't altered if this ends up blank and there is no type of "missing image" icon. A simple grey bg would be fine too.

Reply:
Added a background color. If the sprite or svg fails to load, KvFlag will still take up the space allowed but show a grey box.